dev/CommandBarFlyout/CommandBarFlyout.cpp

```diff
@@ -127,22 +127,41 @@ CommandBarFlyout::CommandBarFlyout()
 
             if (auto commandBar = m_commandBar.get())
             {
-                // If we don't have IFlyoutBase5 available, then we assume a standard show mode.
-                if (!thisAsFlyoutBase5 || thisAsFlyoutBase5.ShowMode() == winrt::FlyoutShowMode::Standard)
-                {
-                    commandBar.IsOpen(true);
-                }
-
+                // If we are in AlwaysExpanded mode then we want to make sure we open in Standard ShowMode
+                // Otherwise the timing on the creation of the two drops shows is such that the primary items
+                // draw their shadow on top of the secondary items.
                 // When CommandBarFlyout is in AlwaysOpen state, don't show the overflow button
                 if (AlwaysExpanded())
                 {
-                    commandBar.IsOpen(true);
                     commandBar.OverflowButtonVisibility(winrt::Windows::UI::Xaml::Controls::CommandBarOverflowButtonVisibility::Collapsed);
+                    if (thisAsFlyoutBase5)
+                    {
+                        thisAsFlyoutBase5.ShowMode(winrt::FlyoutShowMode::Standard);
+                    }
                 }
                 else
                 {
                     commandBar.OverflowButtonVisibility(winrt::Windows::UI::Xaml::Controls::CommandBarOverflowButtonVisibility::Auto);
                 }
+                SharedHelpers::QueueCallbackForCompositionRendering(
```

Comment:
The only reason the timing before this change worked is because the animation that was being used caused a size changed event to occur, which ends up calling UpdateVisuals().  With queueing the callback the timing of events works regardless of the animation type.